runtime/vm/createramclass.cpp

```diff
@@ -2740,6 +2810,14 @@ internalCreateRAMClassFromROMClass(J9VMThread *vmThread, J9ClassLoader *classLoa
 	J9ClassLoader* hostClassLoader = classLoader;
 	J9Module* module = NULL;
 
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+	J9Class *tmpValueTypes[VALUE_TYPE_CLASS_COUNT_ARRAY_THRESHOLD] = {0};
+	J9Class **valueTypes = tmpValueTypes;
+	if (romClass->valueTypeClassCount > VALUE_TYPE_CLASS_COUNT_ARRAY_THRESHOLD) {
+		valueTypes = (J9Class **) calloc(romClass->valueTypeClassCount, sizeof(J9Class *));
+	}
```

Comment:
What if you freed it in this function? 
If we can minimize the places to free that would be better. Makes the code less prone to bugs when refactoring occurs

Reply:
Done.